subsys/mpsl/mpsl_init.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <init.h>
+#include <irq.h>
+#include <kernel.h>
+#include <mpsl.h>
+
+#define MPSL_LOW_PRIO_IRQ SWI5_IRQn
+
+static void m_assert_handler(const char *const file, const u32_t line)
+{
+	k_oops();
+}
+
+static int mpsl_lib_init(struct device *dev)
+{
+	ARG_UNUSED(dev);
+
+	int err = 0;
+	mpsl_clock_lf_cfg_t clock_cfg;
+
+#ifdef CONFIG_CLOCK_CONTROL_NRF_K32SRC_RC
+	clock_cfg.source = MPSL_CLOCK_LF_SRC_RC;
+#elif CONFIG_CLOCK_CONTROL_NRF_K32SRC_XTAL
+	clock_cfg.source = MPSL_CLOCK_LF_SRC_XTAL;
+#elif CONFIG_CLOCK_CONTROL_NRF_K32SRC_SYNTH
+	clock_cfg.source = MPSL_CLOCK_LF_SRC_SYNTH;
+#else
+#error "Clock source is not supported or not defined"
+#endif
+
+	clock_cfg.rc_ctiv = MPSL_RECOMMENDED_RC_CTIV;
+	clock_cfg.rc_temp_ctiv = MPSL_RECOMMENDED_RC_TEMP_CTIV;
+
```

Comment:
clock accuracy is gone.
CLOCK_CONTROL_NRF_ACCURACY Kconfig option does not do anything. Should we remove the option for MPSL?

Reply:
see https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1390/files#diff-2bf5c014915078f1977935e80a88784eR38